flocker/acceptance/test_environment_variables.py

```diff
@@ -247,12 +245,15 @@ def get_mysql_node_2(ignored):
             Move MySQL to ``node_2`` and return a ``Deferred`` which fires
             with a connection to the previously created database on ``node_2``.
             """
+            # Listen on different port so it's clear we're connecting to
```

Comment:
That's a shame. IIRC it used to use Docker and was at some point switched to use internal Flocker code, which we said we could trust enough.